README.md

```diff
@@ -18,6 +18,14 @@ PRISM Break follows [the GNU/FSF definition of Free Software](https://www.gnu.or
 
 PRISM Break strives to promote the best open source applications. Ease of use, stability, and performance matter. This is the first time many people are looking to leave their proprietary walled gardens. Let's make it a good experience for them. If you're writing a privacy-minded FOSS app, please finish it before asking PRISM Break to promote it.
 
+ - **The software must actually be secure.**
```

Comment:
I'd loose the description here, because it's hard to evaluate whether something is actually secure. It's better to say something along these lines: **Must have a strong security focus**